django_netjsonconfig/base/device.py

```diff
@@ -77,7 +77,7 @@ def _get_config_attr(self, attr):
         if not self._has_config():
             return None
         attr = getattr(self.config, attr)
-        return attr() if callable(attr) else attr
```

Comment:
No. This change is not required. Read the comment about callable.

Reply:
I think the if and else statement should be removed right ?

Reply:
No. No reverse this change entirely, it was the `callable(authenticated)`, as mentioned in the link I send you in the comment above.

Reply:
Okay :smile:, got the reason :smile: